src/sentry/api/authentication.py

```diff
@@ -422,7 +423,7 @@ def authenticate_token(self, request: Request, token_str: str) -> tuple[Any, Any
         if application_is_inactive:
             raise AuthenticationFailed("UserApplication inactive or deleted")
 
-        if token.organization_id:
+        if token.scoping_organization_id:
```

Comment:
The `SystemToken` and `ApiTokenReplica` types don't have `scoping_organization_id` attributes. I think you'd need to make sure that attribute exists first.

Reply:
oh got it. Needs to be in a separate PR. Let's do that first.

Reply:
Added both in https://github.com/getsentry/sentry/pull/81213